components/activecampaign/sources/campaign-bounce/campaign-bounce.mjs

```diff
@@ -1,17 +1,18 @@
-const activecampaign = require("../../activecampaign.app.js");
-const common = require("../common-campaign.js");
+import common from "../common/campaign.mjs";
 
-module.exports = {
+export default {
   ...common,
   name: "New Campaign Bounce (Instant)",
   key: "activecampaign-campaign-bounce",
   description:
     "Emits an event when a contact email address bounces from a sent campaign.",
-  version: "0.0.1",
+  version: "0.0.2",
   methods: {
```

Comment:
We need to specify `type: "source"` in each individual source file. If I'm not mistaken, eslint uses it to determine which rules to use.
For `dedupe`, I've been told it's best to put those in the individual source files as well. I'd remove `type` and `dedupe` from `common/base.mjs`, and add them to each source.
```suggestion
  version: "0.0.2",
  type: "source",
  dedupe: "unique",
  methods: {
```